applications/MeshMovingApplication/python_scripts/mesh_solver_base.py

```diff
@@ -60,6 +60,13 @@ def __init__(self, model, custom_settings):
 
         self.settings.ValidateAndAssignDefaults(default_settings)
 
+        if custom_settings["calculate_mesh_velocities"].GetBool():
+            from KratosMultiphysics.kratos_utilities import IssueDeprecationWarning
+            warn_msg  = "The calculation of the Mesh-Velocities is set to ON for the Solver\n"
```

Comment:
here may be we write `calculate_mesh_velocities` is set `true` ?? 
And pointing how to properly calculate mesh velocities (using utility) would also be helpful !

Reply:
ok done